src/web/components/Card/Card.tsx

```diff
@@ -205,6 +211,7 @@ export const Card = ({
 									isFullCardImage={isFullCardImage}
 								>
 									<CardHeadline
+										// Try and create parity with frontend, not working at the moment
```

Comment:
why does it not work? not sure if this comment is very useful

Reply:
That was just a message to myself, forgot to take it out

Reply:
This comment is still here 😉